homeassistant/components/imap/coordinator.py

```diff
@@ -50,6 +122,7 @@ def __init__(
     ) -> None:
         """Initiate imap client."""
         self.imap_client = imap_client
+        self._last_mesgid: str | None = None
```

Comment:
Let's write this out.
```suggestion
        self._last_message_id: str | None = None
```